build_tools/python/benchmark_suites/iree/vulkan_nvidia_benchmarks.py

```diff
@@ -88,15 +88,15 @@ def generate(
   ) -> Tuple[List[iree_definitions.ModuleGenerationConfig],
              List[iree_definitions.E2EModelRunConfig]]:
     """Generates IREE compile and run configs."""
-    # The `vulkan-nvidia` tag is required to put them into the Vulkan NVIDIA
+    # The `vulkan-nvidia`` tag is required to put them into the Vulkan NVIDIA
```

Comment:
```suggestion
    # The `vulkan-nvidia` tag is required to put them into the Vulkan NVIDIA
```

Revert extra backtick?